enhancements/audit-command.md

```diff
@@ -0,0 +1,722 @@
+
+The downside of create a new tool for this purpose shows mainly be the effort to keep it maintained and not take advantage of is  implemented already as ask for users which who requires to audit the OLM index catalogs and would like to use this feature also install an additional tool since they probably work already with opm or sdk and maybe both indeed. 
+
+**Conclusion**
+//TODO: Add the consensus reached out. 
+
+
+## Summary
+
+This proposal describes a new operation provided via an alpha command/solution which has the purpose to audit all bundles of some specific OLM index catalog and then, output a report.
+
+## Motivation
+
+### Goals
+
+- Be able to audit all bundles of an OLM index catalog and output a report
+- Be able to easily check and identify aspects of the operators bundles which will be published on it such as:
+    - check if they are using deprecated apis
+    - check if they are built with sdk and its versions
+    - the language used 
+    - check if they are multi-arch
+    - check if they are using some relevant feature provided by looking their annotations
+    - check if they are following the recommendations and conventions defined
+    - check all warnings and errors which will be raised by using the SDK features scorecard and bundle validate
+- Be able to extract a report with the audit results and in some formats such as json. 
+- Be able to run specific custom checks for each bundle in order to audit the catalog
+- Provide a documentation over how to use the audit command
+
+
+### Non-Goals
+
+- Add new checks or validators. This proposal has the goal to define the command and operation only. The checks should be addressed in their respective proposals.
+- Allow users select what specific validators will be used by the SDK commands. However, it can be addressed in the future.
+- Provide documentation that will describe how to add the custom scorecard tests in the bundle done with [Writing Custom Scorecard Tests](https://sdk.operatorframework.io/docs/advanced-topics/scorecard/custom-tests/) manually for Non-SDK projects or bundles which are not built with `make bundle` and how they are used with the audit command.
+ 
+**IMPORTANT**: Note that we are describing here an analytic tool/command that will start to be implemented in **alpha**. This feature uses what is in place and implemented already and cover needs which are raised in other proposals such as [Custom operator validation #43](https://github.com/operator-framework/enhancements/pull/43). However, it does not make any change or requires any new behaviour for the operator authors. The possibilities covered by the audit command can be easily documented after the first version of it this command be in place. 
+
+## Proposal
+
+Add a new command `audit` with the purpose to audit all desired operator bundle data of a specific index catalog. 
```

Comment:
Add a new command to WHAT? It is not clear if the plan is to add it to `opm` or to `operator-sdk`. I would suggest that you pick one approach in the main portion of this EP. Then document the other approach in an `Alternative` section in the EP. That way both can be documented.

Reply:
See the open question. I clarified that. It is not the goal of the EP defined that. It is secundary.